f5/bigip/tm/cm/trust.py

```diff
@@ -54,8 +58,12 @@ class Remove_From_Trust(UnnamedResourceMixin, CommandExecutionMixin, Resource):
 
     def __init__(self, cm):
         super(Remove_From_Trust, self).__init__(cm)
+        base_uri = type(self).__name__.replace('_', '-').lower()
```

Comment:
Here is the error and command used to invoke it:

https://gist.github.com/wojtek0806/249f35ac98c4a636779eee5fdb9edec8#file-gistfile1-txt